pie/torch_utils.py

```diff
@@ -228,11 +228,14 @@ def word_dropout(inp, p, training, encoder):
           for j in range(batch)] for i in range(seq_len)]
     ).float().to(inp.device)
     # compute bernoulli mask
-    mask = 1 - torch.bernoulli(mask / (p + mask))
+    # PyTorch 1.2 and 1.3 :
+    #   - Deprecated masked_fill_ used with int.
+    #   - Mask should be booleans in general
+    mask = ((1 - torch.bernoulli(mask / (p + mask))) == 1)
```

Comment:
I find this a bit hard to read. Couldn't we just do:

```
mask = 1 - torch.bernoulli(mask / (p + mask))
mask = mask.byte()
```

Reply:
mask.byte() is actually the issue, should have used bool but did not see it in the doc...